plugin/src/main/java/com/denizenscript/denizen/nms/interfaces/EntityHelper.java

```diff
@@ -408,71 +372,31 @@ public void clientResetLoc(Entity entity) {
 
     public abstract void teleport(Entity entity, Location loc);
 
-    public abstract BoundingBox getBoundingBox(Entity entity);
+    public abstract void setBoundingBox(Entity entity, BoundingBox box);
 
-    public abstract void setBoundingBox(Entity entity, BoundingBox boundingBox);
-
-    public List<Player> getPlayersThatSee(Entity entity) {
-        throw new UnsupportedOperationException();
-    }
+    public abstract List<Player> getPlayersThatSee(Entity entity);
 
     public void sendAllUpdatePackets(Entity entity) {
         throw new UnsupportedOperationException();
     }
 
-    public void setTicksLived(Entity entity, int ticks) {
-        entity.setTicksLived(ticks);
-    }
+    public abstract void setTicksLived(Entity entity, int ticks);
 
-    public int getShulkerPeek(Entity entity) {
-        throw new UnsupportedOperationException();
-    }
-
-    public void setShulkerPeek(Entity entity, int peek) {
-        throw new UnsupportedOperationException();
-    }
-
-    public void setHeadAngle(Entity entity, float angle) {
-        throw new UnsupportedOperationException();
-    }
+    public abstract void setHeadAngle(Entity entity, float angle);
 
-    public void setGhastAttacking(Entity entity, boolean attacking) {
+    public void setGhastAttacking(Entity entity, boolean attacking) { // TODO: 1.19 - Ghast#setCharging

```

Comment:
If this has API in 1.19, remove the `UnsupportedOperationException` and remove the 1.19 impl, and make the interface impl use the API method (ie so old versions use NMS and new versions don't, and so we know more easily this is ready to remove once there are no implementing modules)

Reply:
Should be fixed